src/seedu/addressbook/parser/Parser.java

```diff
@@ -97,7 +97,10 @@ public Command parseCommand(String userInput) {
         case ExitCommand.COMMAND_WORD:
             return new ExitCommand();
 
-        case HelpCommand.COMMAND_WORD: // Fallthrough
+        case SortingCommand.COMMAND_WORD:
```

Comment:
Does this naming follow the naming convention? Perhaps "SortCommand" would be better!